functions.sh

```diff
@@ -3296,12 +3296,12 @@ report_statistic() {
   if [ "$1" -a "$2" -a "$3" -a "$4" -a "$5" ]; then
     REPORTSRV="$1"
 
-    STANDARDIMAGE="`ls -1 "$IMAGESPATH" |grep "$2"`"
+    STANDARDIMAGE="$(ls -1 "$IMAGESPATH" |grep "$2")"
```

Comment:
$2 is the second argument of the function. The original function does an ls on a directory and greps for $2..

The reworked function (from my comment) does exactly the same with bashglobbing. ($2 is supposed to be surrounded with '*' which was interpreted as some markdown formating.)